spec/index.md

```diff
@@ -504,9 +502,7 @@ Whilst strictly not an entity, the concept of an "account" from which money is s
 
 ### Location
 
-There is a frequent desire to label items with location, usually by attaching geographic codes for a region or area. This allows the spending or revenue to  be analysed by region or area. This geographic information can be introduced directly by classifying the item with a code, or, more frequently indirectly by associating a geographic code to e.g. an entity. For example, by labelling a supplier with their location one can then associate a spend with that supplier as spending in that location.
```

Comment:
We have lost here the point about labelling entities with location vs the budget lines themselves. Are we saying this is a distinction not worth making (quite possible btw ...)?